business/openid_auth.go

```diff
@@ -491,9 +464,9 @@ func RequestOpenIdToken(openIdParams *OpenIdCallbackParams, redirect_uri string)
 		}
 	}
 
-	httpClient := http.Client{
-		Timeout:   time.Second * 10,
-		Transport: httpTransport,
```

Comment:
is the `httpTransport` variable unused now?

Reply:
Yes, just removed it, thanks!